Week2/roundAndCompare.js

```diff
@@ -0,0 +1,11 @@
+console.log(`------ 5 ------`);
+
+const z = 7.25;
+console.log(`value of z: ${z}`);
+
+const a = Math.round(z);
+console.log(`value of a: ${a}`);
+
+const highest = Math.max(z , a);
```

Comment:
Nice solution!

Reply:
Hello Marcia. Thank you very much for the feedback.